backend/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:8-jdk-alpine
+#ARG JAR_FILE=target/demo-app-1.0.0.jar
+ARG JAR_FILE=./build/libs/demo-0.0.1-SNAPSHOT.jar
+ADD ${JAR_FILE} app.jar

```

Comment:
Hello @XIC557-RamaKrishna thank you for your submission. :)

I am part of the team who's going to review your code.

We tried to run this project with the aforementioned command `docker-compose up` but it won't run, for the reasons highlighted in this Dockerfile. Can you fix this in order to abide to the challenge requirements?

Thanks!